ci-operator/step-registry/openshift-qe/kube-burner/openshift-qe-kube-burner-chain.yaml

```diff
@@ -0,0 +1,8 @@
+
+chain:
+  as: openshift-qe-kube-burner
+  steps:
+  - ref: openshift-qe-kube-burner
+  - ref: openshift-qe-write-perfscale-results
```

Comment:
the write job has been deprecated so you can take it out here

Reply:
depending on when the cerberus check goes in you'll have to update this chain: https://github.com/openshift/release/pull/42580